src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue8609Tabbed.cs

```diff
@@ -0,0 +1,23 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue8609Tabbed : _IssuesUITest
+	{
+		public Issue8609Tabbed(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "TabbedPage.AutoResizeIconsProperty";
+
+		[Test]
+		[Category(UITestCategories.TabbedPage)]
+		public void TabBarIconsShouldAutoscale()
+		{
+			App.WaitForElement("Tab1");
+			VerifyScreenshot();
```

Comment:
Pending snapshots. Could you commit the images?

Example, Windows:
![image](https://github.com/user-attachments/assets/8683cb3a-9795-432b-9ec6-4dddbbeea050)